common/persistence/tests/sqlite_test.go

```diff
@@ -1187,7 +1187,7 @@ func TestSQLiteTransactionContextCancellation(t *testing.T) {
 	cancel()
 
 	err = tx.Commit()
-	assert.ErrorAs(t, err, &context.Canceled)
```

Comment:
This yields a data race when the assertion library is trying to wrap the error.

Reply:
How this could possible work at all?

Reply:
🤷